fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/world/poi/PointOfInterestHelper.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.object.builder.v1.world.poi;
+
+import java.util.function.Predicate;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Sets;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.world.poi.PointOfInterestType;
+
+import net.fabricmc.fabric.mixin.object.builder.PointOfInterestTypeAccessor;
+
+/**
+ * This class provides utilities to create a {@link PointOfInterestType}.
+ */
+public final class PointOfInterestHelper {
+	private PointOfInterestHelper() {
+		throw new IllegalStateException("You shouldn't be initializing this!");
+	}
+
+	/**
+	 * Registers and creates a {@link PointOfInterestType}.
```

Comment:
The description could help disambiguate the various overloads by telling what is special about each.

Blank line after the 1st sentence and/or before the tag area please.